src/components/Containers/UASLayout/UASLayout.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Text } from "../../Texts";
+import { baseTheme } from "../../../theme";
+import { media } from "../../../utils";
+
+export const UASLayout = ({ titleText, graphic, descriptionText, ...props }) => (
+  <div {...props}>
+    <TitleSection>
+      <Title size={baseTheme.size.h2} bold="true">
+        {titleText}
+      </Title>
+    </TitleSection>
+    <InfoSection>
+      <Graphic src={graphic} />
+      <Description>{descriptionText}</Description>
+    </InfoSection>
+  </div>
+);
+
+const Title = styled(Text)`
+  ${({ theme }) => `
+      font-family: ${theme.font.josefin};
+      color: ${theme.colors.black};
+  `};
+`;
+
+const Description = styled(Text)`
+  flex: 1;
+`;
+
+const Graphic = styled.img`
+  width: 300px;
+  height: 240px;
```

Comment:
In future projects, defining a set width and height is risky but it will not matter too much for this project since ```Next.js``` has an image optimization feature we can use, but just thought I would heads up this.